README.adoc

```diff
@@ -230,10 +230,35 @@ single master node you can disable loadbalancer node by including
 heat stack-create my_openshift \
    -e openshift_parameters.yaml \
    -P master_count=1 \
-   -f openshift-on-openstack/openshift.yaml\
+   -f openshift-on-openstack/openshift.yaml \
    -e env_loadbalancer_none.yaml
 ```
 
+== Select SDN Type ==
+
+By default, OpenShift is deployed with https://docs.openshift.com/enterprise/3.2/architecture/additional_concepts/sdn.html[OpenShift-SDN].
+When used with Openstack Neutron with GRE or VXLAN tunnels, packets are encapsulated twice
+which can have an impact on performances. Those Heat templates allow using https://github.com/coreos/flannel[Flannel]
+instead of openshift-sdn, with the `host-gw` backend to avoid the double encapsulation.
+To do so, you need to include the `env_flannel.yaml` environment file when you create the stack:
+
+```bash
+heat stack-create my_openshift \
+   -e openshift_parameters.yaml \
+   -f openshift-on-openstack/openshift.yaml \
+   -e env_sdn_flannel.yaml
```

Comment:
shouldn't it be "openshift-on-openstack/env_flannel.yaml" (according to the fact that newely added env file is named env_flannel.yaml?